src/validate.js

```diff
@@ -1,7 +1,28 @@
 import REGEX from './regex.js';
+import version from './version.js';
 
 function validate(uuid) {
   return typeof uuid === 'string' && REGEX.test(uuid);
 }
 
 export default validate;
```

Comment:
I think it makes sense to use only named exports in this module.

Reply:
not sure what you mean?
Most other things seem to be done as default exports.

Reply:
They don't have named exports. Validate is not that special to use different export syntax.

Reply:
Ahhh, I understand what you mean now. Good point.
I agree that'd be more consistent with the rest of the source. I'll revise my branch.